File: src/CENTRO_SALUD/RecetaMedica.java
package CENTRO_SALUD;

import java.util.ArrayList;
import java.util.List;

public class RecetaMedica {
    private String idReceta;
    private String fecha;
    private List<String> medicamentos = new ArrayList<>();
    private String indicaciones;
    private Atencion atencion; // atencion que origina la receta

    public RecetaMedica(String idReceta, String fecha, String indicaciones, Atencion atencion) {
        this.idReceta = idReceta;
        this.fecha = fecha;
        this.indicaciones = indicaciones;
        this.atencion = atencion;
    }

    public void agregarMedicamento(String medicamento) {
        medicamentos.add(medicamento);
    }

    public String getIdReceta() { return idReceta; }
    public String getFecha() { return fecha; }
    public List<String> getMedicamentos() { return medicamentos; }
    public String getIndicaciones() { return indicaciones; }
    public Atencion getAtencion() { return atencion; }

    @Override
    public String toString() {
        return "Receta " + idReceta + " (" + fecha + ") - " + medicamentos.size() + " medicamento(s)";
    }
}

File: centro_salud_python/receta_medica.py
class RecetaMedica:
    """Receta medica que se genera a partir de una atencion."""

    def __init__(self, id_receta, fecha, indicaciones, atencion):
        self._id_receta = id_receta
        self._fecha = fecha
        self._medicamentos = []
        self._indicaciones = indicaciones
        self._atencion = atencion  # atencion que origina la receta

    def agregar_medicamento(self, medicamento):
        self._medicamentos.append(medicamento)

    def get_id_receta(self):
        return self._id_receta

    def get_fecha(self):
        return self._fecha

    def get_medicamentos(self):
        return self._medicamentos

    def get_indicaciones(self):
        return self._indicaciones

    def get_atencion(self):
        return self._atencion

    def __str__(self):
        return f"Receta {self._id_receta} ({self._fecha}) - {len(self._medicamentos)} medicamento(s)"
